packages/plugin-commands-audit/src/fix.ts

```diff
@@ -19,10 +19,10 @@ export async function fix (dir: string, auditReport: AuditReport) {
   return vulnOverrides
 }
 
-function createOverrides (advisories: AuditAdvisory[]) {
+function createOverrides (advisories: AuditAdvisory[], allowList?: string[]) {
   return fromPairs(
-    advisories
-      .filter(({ vulnerable_versions, patched_versions }) => vulnerable_versions !== '>=0.0.0' && patched_versions !== '<0.0.0') // eslint-disable-line
+    advisories.filter(({ cves }) => allowList ? !allowList.join(',').includes(cves.join(',')) : true)
```

Comment:
This doesn't seem right. Shouldn't it be something like
```suggestion
    advisories.filter(({ cves }) => allowList ? difference(allowList, cves).length === 0 : true)
```

Reply:
Tweaked slightly what you provided since it wasn't working for me, but using `difference` does make this more readable, thanks!